tests/test_backup.py

```diff
@@ -8,26 +8,36 @@
 NOPREV_MSG = ("ERROR: option '--incremental': Previous backup "
               "directory does not exist")
 
-OFFLINE_BACKUP_FILES = [
+OFFLINE_CAPS_FILES = [
     'config_files.tar.gz',
     '.config.snar',
     'metadata.yml',
     'mongo_data.tar.gz',
     '.mongo.snar',
+]
+
+OFFLINE_SAT_FILES = [
     'pgsql_data.tar.gz',
     '.postgres.snar',
 ]
 
-ONLINE_BACKUP_FILES = [
-    'candlepin.dump',
+OFFLINE_BACKUP_FILES = OFFLINE_CAPS_FILES + OFFLINE_SAT_FILES
```

Comment:
@pondrejk I didn't get why we need combined list of satellite and capsule here?

Reply:
Can we just keep 4 different list such as ONLINE_CAPS_FILES,ONLINE_SAT_FILES,OFFLINE_CAPS_FILES and OFFLINE_SAT_FILES. And assert according to just those sets?

Reply:
yeah, it's not really necessary, will update

Reply:
ah ok, now I remember, the idea was to avoid a bit of duplication, so for example, if mongo_data.tar.gz gets renamed we would have to update it just once instead of twice (both ONLINE_CAPS_FILES and ONLINE_SAT_FILES), @ntkathole what do you think?